organizations/user.go

```diff
@@ -145,7 +145,7 @@ func CreateMember(w http.ResponseWriter, r *http.Request) {
 	memDoc, _ := utils.GetMongoDbDocs(member_collection, bson.M{"org_id": sOrgId, "email": newUserEmail})
 	if memDoc != nil {
 		fmt.Printf("organization %s has member with email %s!", orgId.String(), newUserEmail)
-		utils.GetError(errors.New("user is already in this organization"), http.StatusBadRequest, w)
+		utils.GetError(errors.New("User is already in this organisation"), http.StatusBadRequest, w)
```

Comment:
The uppercase U you changed is illegal according to go-checker, it works like eslint in javascript. Syntactically, it's right, but it's not a good practice according to go-checker. and you should avoid making changes that do not impact your work.